src/OrleansServiceBus/Providers/Streams/EventHub/EventHubPartitionCheckpoint.cs

```diff
@@ -0,0 +1,74 @@
+﻿
+using System;
+using System.Threading.Tasks;
+using Microsoft.ServiceBus.Messaging;
+using Orleans.AzureUtils;
+
+namespace Orleans.ServiceBus.Providers
+{
+    internal class EventHubPartitionCheckpoint
```

Comment:
I agree.  This was not a dependency I was happy with.  In the future an injectable checkpoint would be preferred, but I want to give some time for this to bake before further consideration.

Will comment.
Not sure about moving, but will definitely consider it.